k9mail/src/main/java/com/fsck/k9/Preferences.java

```diff
@@ -164,4 +166,22 @@ public void setDefaultAccount(Account account) {
     public SharedPreferences getPreferences() {
         return mStorage;
     }
+
+    public static <T extends Enum<T>> T getEnumStringPref(SharedPreferences prefs, String key, T defaultEnum) {
+        String stringPref = prefs.getString(key, defaultEnum.name());
+
+        if (null == stringPref) {
```

Comment:
I'm done with this merge request. If you guys want to make further fixes, go ahead. Or just close it.